backend/config/settings/gcp.py

```diff
@@ -8,4 +8,11 @@
 DJANGO_DRF_FILEPOND_STORAGES_BACKEND = "storages.backends.gcloud.GoogleCloudStorage"
 GS_BUCKET_NAME = env("BUCKET_NAME", "doccano")
 GS_PROJECT_ID = env("GS_PROJECT_ID")
-GS_CREDENTIALS = service_account.Credentials.from_service_account_file(env("GOOGLE_APPLICATION_CREDENTIALS"))
+
+# for more details refer to
+# https://django-storages.readthedocs.io/en/latest/backends/gcloud.html#authentication-settings
+_google_application_credentials = env("GOOGLE_APPLICATION_CREDENTIALS", "")
+if _google_application_credentials:
+    GS_CREDENTIALS = service_account.Credentials.from_service_account_file(
+        _google_application_credentials
+    )
```

Comment:
Perhaps we ought to set the default to None when not set:


```suggestion
_google_application_credentials = env("GOOGLE_APPLICATION_CREDENTIALS", "")
if _google_application_credentials:
    GS_CREDENTIALS = service_account.Credentials.from_service_account_file(
        _google_application_credentials
    )
else:
    GS_CREDENTIALS = None
```